docs/release-notes/release-notes-0.14.0.md

```diff
@@ -7,4 +7,11 @@ added](https://github.com/lightningnetwork/lnd/pull/5520) to ensure that all
 PRs add an entry in the release notes folder that at leasts links to PR being
 added.
 
+# Code Health
+
+## Code cleanup, refactor, typo fixes
+* [Unused error check removed] 
```

Comment:
One last nit :sweat_smile:: This doesn't seem to render correctly: https://github.com/ErikEk/lnd/blob/8da921732de13b8e2b39683dedc764965df02001/docs/release-notes/release-notes-0.14.0.md

I think it should be:
```
* [Unused error check
   removed](...)
```